Rakefile

```diff
@@ -5,3 +5,13 @@
 require File.expand_path('../config/application', __FILE__)
 
 SmartAnswers::Application.load_tasks
+
+# Delete the current "default" rake task and redefine it. This allow us to use:
+# - `rake test` to only run minitest tests
+# - `rake spec` or `rspec` to only run rspec tests
+# - `rake` to run both minitest and rspec tests
+Rake::Task["default"].clear
+task :default do
+  system("bundle exec rake spec")
```

Comment:
How does this handle failures, will the default task exit with a non 0 exit code if either of the sets of tests fail?